playbook.yml

```diff
@@ -2,13 +2,3 @@
 - import_playbook: setup_eap.yml
 - import_playbook: prepare_migration.yml
 - import_playbook: perform_migration.yml
-
```

Comment:
Why did you remove the 7.3 to 7.4 demo?

Reply:
we are already doing that in our demo from EAP 7.3 to EAP 7.4, so removed that part.